spec/controllers/holds_controller_spec.rb

```diff
@@ -148,5 +148,23 @@
         end
       end
     end
+
+    describe '#new' do
+      let(:bib) { build(:bib_with_holdables) }
+      let(:response_body) { HOLDABLE_LOCATIONS_RAW_JSON }
+
+      before do
+        stub_request(:get, 'https://example.com/symwsbc/catalog/bib/key/1?includeFields=*,callList%7B*,itemList%7B*%7D%7D')
+          .to_return(status: 200, body: bib.body.to_json, headers: {})
```

Comment:
@cdmo `.with(query: hash_including(includeFields:''))` here as well? Works the same day but just a bit more readable? what do you think?

Reply:
yes